src/Elcodi/Plugin/StoreSetupWizardBundle/Resources/translations/messages.en.yml

```diff
@@ -7,7 +7,7 @@ wizard:
     go_enable_store: Go to setup and disable the "under construction" mode.
     go_next_step: Go to next step
     step_finished: <strong>Yehaaaaa!</strong> One step less to go!
-    time_to_sell: You are <strong>%1 minutes</strong> away to start selling
+    time_to_sell: You are <strong>%1 minutes</strong> away from start selling
```

Comment:
I'm not sure, but I think the idiom is *"X minutes away from starting ..."* instead of *"X minutes away from start..."*.

Just a suggestion: why not replacing it by: *"<strong>%1 minutes</strong> left to opening your store"*